### DISCUSSION TITLE
Repo Maintained?

### DISCUSSION THREAD

**Discussion by rareplanet1** [sashaskr/pinpayments-go#10] [Q&A]
Hi @sashaskr 
Thanks for this project.
Is is still being maintained?

**Comment by sashaskr** [comment]
Hi @rareplanet1, yes, as far Pinpayments introduces the new functionality once I have a free time - I add it.

**Reply by rareplanet1** [reply]
Thanks for your reply.
I've started a fork in the meantime if you want to pull in any changes.
https://github.com/rareplanet1/pinpayments-go
